packages/mobile/src/notifications/NotificationList.tsx

```diff
@@ -0,0 +1,95 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import variables from '@celo/react-components/styles/variables'
+import React, { useEffect } from 'react'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import SafeAreaView from 'react-native-safe-area-view'
+import {
+  NavigationParams,
+  NavigationProp,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation'
+import { CURRENCIES, CURRENCY_ENUM } from 'src/geth/consts'
+import i18n from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import DisconnectBanner from 'src/shared/DisconnectBanner'
+import { getCentAwareMoneyDisplay } from 'src/utils/formatting'
+
+const { contentPadding } = variables
+
+interface OwnProps<T> {
+  dollarBalance: string | null
+  listItemRenderer: (item: T, key: number) => JSX.Element
+  items: T[]
+}
+
+type Props<T> = OwnProps<T>
+
+export function NotificationList<T>(props: Props<T>) {
+  return (
+    <SafeAreaView style={styles.container}>
+      <DisconnectBanner />
+      {props.items.length > 0 ? (
+        <ScrollView>
+          <View style={[styles.scrollArea]}>{props.items.map(props.listItemRenderer)}</View>
+        </ScrollView>
+      ) : (
+        <Text style={[fontStyles.bodySecondary, styles.empty]}>{i18n.t('global:emptyList')}</Text>
```

Comment:
Ditto feedback as above for use of i18n here

Reply:
Also, is 'Empty List' all the design calls for here?

Reply:
There is only one call to `t`, is that advisable to still use `withNamespace`?
Can you rephrase "is 'Empty List' all the design calls for here" - I did not get that :)

Reply:
Yes please, still use withNamespace

Nevermind the Empty List bit for now. Was just a comment on the intended design. I think showing the words "Empty List" is suboptimal but fine for now